### DISCUSSION TITLE
Replay .mkv video using Plus Server

### DISCUSSION THREAD

**Discussion by lbo2021** [PlusToolkit/PlusLib#887] [Q&A]
Hi,

I recorded a video in .mkv format using PlusServerRemoteControl.exe.

Now I would like to replay this video using Plus Server. 

My configuration file is the one attached to this comment, but it does not work
[Mkv_Video_Replay.zip](https://github.com/PlusToolkit/PlusLib/files/7854076/Mkv_Video_Replay.zip)

In the Plus Server log files I can read:

`|INFO|6035.589000| Connect using configuration file: C:\Users\z004e4fu\PlusApp-2.8.0.20191105-Win64\config\Mkv_Video_Replay.xml| in E:\D\PSNP64b\PlusApp\PlusServerLauncher\PlusServerLauncherMainWindow.cxx(639)
|INFO|6035.590000| Server process command line: "C:/Users/z004e4fu/PlusApp-2.8.0.20191105-Win64/bin/PlusServer.exe" --config-file="Mkv_Video_Replay.xml" --verbose=3| in E:\D\PSNP64b\PlusApp\PlusServerLauncher\PlusServerLauncherMainWindow.cxx(320)
|INFO|6036.232000| Server process started successfully| in E:\D\PSNP64b\PlusApp\PlusServerLauncher\PlusServerLauncherMainWindow.cxx(329)
|INFO|6036.233000|SERVER> System start timestamp: 433196
|INFO|6036.234000|SERVER> Software version: Plus-2.8.0.62873a16 - Win64
|INFO|6036.234000|SERVER> Logging at level 3 (INFO) to file: C:/Users/z004e4fu/PlusApp-2.8.0.20191105-Win64/data/011222_115953_PlusLog.txt
|INFO|6036.234000|SERVER> Server status: Reading configuration.
|INFO|6036.234000|SERVER> Server status: Connecting to devices.
|INFO|6037.482000|SERVER> Copy buffer to video buffer...
|WARNING|6037.483000|SERVER> Failed to add video frame to buffer from sequence metafile with frame #1| in :\D\PSNP64b\PlusLib\src\PlusDataCollection\vtkPlusBuffer.cxx(992)
|WARNING|6037.501000|SERVER> Failed to add video frame to buffer from sequence metafile with frame #2| in :\D\PSNP64b\PlusLib\src\PlusDataCollection\vtkPlusBuffer.cxx(992)
|WARNING|6037.501000|SERVER> Failed to add video frame to buffer from sequence metafile with frame #3| in :\D\PSNP64b\PlusLib\src\PlusDataCollection\vtkPlusBuffer.cxx(992)
|WARNING|6037.501000|SERVER> Failed to add video frame to buffer from sequence metafile with frame #4| in :\D\PSNP64b\PlusLib\src\PlusDataCollection\vtkPlusBuffer.cxx(992)
|WARNING|6037.501000|SERVER> Failed to add video frame to buffer from sequence metafile with frame #5| in :\D\PSNP64b\PlusLib\src\PlusDataCollection\vtkPlusBuffer.cxx(992)
|WARNING|6037.501000|SERVER> Failed to add video frame to buffer from sequence metafile with frame #6| in :\D\PSNP64b\PlusLib\src\PlusDataCollection\vtkPlusBuffer.cxx(992)
|WARNING|6037.502000|SERVER> Failed to add video frame to buffer from sequence metafile with frame #7| in :\D\PSNP64b\PlusLib\src\PlusDataCollection\vtkPlusBuffer.cxx(992)
|WARNING|6037.502000|SERVER> Failed to add video frame to buffer from sequence metafile with frame #8| in :\D\PSNP64b\PlusLib\src\PlusDataCollection\vtkPlusBuffer.cxx(992)
|WARNING|6037.502000|SERVER> Failed to add video frame to buffer from sequence metafile with frame #9| in :\D\PSNP64b\PlusLib\src\PlusDataCollection\vtkPlusBuffer.cxx(992)`

Can you please tell me how to replay a .mkv video?

Thank you

**Comment by Sunderlandkyl** [comment]
What codec is the MKV encoded with? Currently Plus only supports VP9.
